api/can_api_v2_definition.py

```diff
@@ -189,12 +189,21 @@ class AnomalyAnnotation(base_model.APIBaseModel):
     )
 
 
+class FieldSource(base_model.APIBaseModel):
+    type: Optional[FieldSourceType] = pydantic.Field(
+        None, description="The type of data source from a CAN list of data source types"
```

Comment:
Should one of these be required? It seems incorrect that a field source annotation would have all 3 None.

Reply:
Agreed that I'd expect at least one to be set but any of them can be None. I'm not sure what to do.